packages/operator-filter/README.md

```diff
@@ -0,0 +1,75 @@
+# <PACKAGE>
+
+*Include a high level description of your package here*
+
+This example project is based on the template generated by hardhat when run in an empty directory.
+
+This project demonstrates a basic Hardhat use case. It comes with a sample contract, a sample test fixture for that contract,
+and tests.
+
```

Comment:
Just need to update the above. It's ok to refer to other documents

Reply:
removed it willl add a proper read me in the pr for docs.

Reply:
This is fine, as long as you retain the information on package usage.